File: estore-api/src/main/java/com/estore/api/estoreapi/model/Order.java
package com.estore.api.estoreapi.model;

import java.util.List;

public class Order {
    private String username;
    private List<Shoe> items;
    private double totalCost;

    public Order (String username, List<Shoe> items, double totalCost) {
        this.username = username;
        this.items = items;
        this.totalCost = totalCost;
    }

    public String getUsername() {
        return this.username;
    }
}

File: estore-ui/src/app/Order.ts
import {Shoe} from "./ShoeInterface";
import {CartService} from "./services/cart.service";
import {Observable} from "rxjs";
import {User} from "./User";

export class Order {
	user: User = new User();
	items: Shoe[] = [];
	totalCost: number = 0;

	constructor(user: User, items: Shoe[], totalCost: number) {
		this.user = user;
		this.items = items;
		this.totalCost = totalCost;
	}

	getUsername(): string {
		return this.user.getUsername()
	}
	getItems(): Shoe[] {
		return this.items;
	}
	getTotalCost(): number {
		return this.totalCost;
	}
}
